users/tests/test_views.py

```diff
@@ -52,9 +52,11 @@ def test_change_user_password_view_should_not_change_user_password_when_new_pass
         response = self.client.post(reverse("password_change"), data)
         self.user.refresh_from_db()
         self.assertEqual(response.status_code, 200)
-        self.assertTrue(
-            ValidationErrorText.VALIDATION_ERROR_SIGNUP_PASSWORD_MESSAGE
-            in response.context["form"].errors.get("new_password2")
+        # Replacement between "’" and "'" it is caused because in response
+        # we got "’" but from strings hardcoded in `strings.py` we got "'"
```

Comment:
And this happens after the Django update? What the heck is this?

Reply:
Yea, I didn't want to dig deep into django to find what exactly changed this is only about apostrophe :/

Reply:
Yeah, I hear you, but it's so baffling! Should we add this to Jira to investigate later maybe?

Reply:
Sure, please add it!